sdk/openai/openai/review/openai-types.api.md

```diff
@@ -146,6 +191,7 @@ export interface ContentFilterResultOutput {
 
 // @public
 export interface ContentFilterResultsForChoiceOutput {
+    custom_blocklists?: ContentFilterDetailedResults;
```

Comment:
It seems like we're missing this new property called `ungroundedMaterial` in [TypeSpec file](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/cognitiveservices/OpenAI.Inference/models/completions/azure_content_filtering.tsp#L142-L146) and here in [inference file](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/cognitiveservices/data-plane/AzureOpenAI/inference/preview/2024-10-01-preview/inference.yaml#L2079C13-L2080), which have 2 new models